### PR TITLE
FEATURE: EGL-387 prepare english texts for localization

### PR DESCRIPTION
**Pull request by phong-nguyen-duy**
JIRA: EGL-387

---

Supported PR commands:

| Command                                                 | Description                                                |
| ------------------------------------------------------- | ---------------------------------------------------------- |
| `ok to test`                                            | Re-run standard checks                                     |
| `extended check sonar`                                  | SonarQube tests                                            |
| `extended test - backstop`                              | BackstopJS tests                                           |
| **E2E Cypress tests commands - TIGER**                  |                                                            |
| `extended test - tiger-cypress - isolated <testName>`   | Run isolated tests running against recorded Tiger backend. |
| `extended test - tiger-cypress - record <testName>`     | Create a new recording for isolated Tiger tests.           |
| `extended test - tiger-cypress - integrated <testName>` | Run integrated tests against live backend                  |
| **E2E Cypress tests commands - BEAR**                   |                                                            |
| `extended test - cypress - isolated <testName>`         | Run isolated tests running against recorded Bear backend.  |
| `extended test - cypress - record <testName>`           | Create a new recording for isolated Bear tests.            |
| `extended test - cypress - integrated <testName>`       | Run integrated tests against live backend                  |
| **Compatibility matrix test commands - TIGER Backend**  |                                                            |
| `extended test - matrix-test <AIO_version>`             | Run integrated tests against AIO versions.                 |

`<testName>` in cypress commands is used to filter specfiles. Example, to run record with BEAR backend

-   Against `dashboard.spec.ts` and `drilling.spec.ts`, execute command `extended test - cypress - record dashboard,drilling`
-   Against all specfiles, execute command `extended test - cypress - record` or `extended test - cypress - record *`

`<AIO_version>` in commands is used to start test with multiple AIO instances - each instance in triggered by one jenkins build

-   To run with `master` and `stable`, execute command `extended test - matrix-test master,stable` or `extended test - matrix-test latest`
-   To run with specific version,ex: `2.3.0` and `2.3.1`, execute command `extended test - matrix-test 2.3.0,2.3.1`
-   In case `<AIO_version>` is empty, read versions from file `compTigerVersions.txt` of this repo

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `extended test - backstop` passes
-   [ ] `extended test - tiger-cypress - record` to record new mapping files (Tiger BE)
-   [ ] `extended test - cypress - record` to record new mapping files (Bear BE)
-   [ ] `extended test - tiger-cypress - isolated` passes
-   [ ] `extended test - cypress - isolated` passes
-   [ ] `extended test - tiger-cypress - integrated` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/16750/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/18194/ : FAILURE in 12m 24s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/16755/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/18199/ : SUCCESS in 12m 06s

File: libs/sdk-ui/src/base/localization/bundles/en-US.json [hunk 1]
```diff
@@ -2015,5 +2015,190 @@
...
         "limit": 0
+    },
+    "configurationPanel.drillConfig.learn.more.link": {
+        "value": "Learn more",
+        "comment": "Link with additional info",
+        "limit": 0
+    },
+    "configurationPanel.drillConfig.drillToDashboard.tooltip": {
+        "value": "Open a related dashboard for an expanded view on the selected data point, carrying over relevant filters and context.",
+        "comment": "",
```
Comment (khanhhuynh-gooddata) [inline libs/sdk-ui/src/base/localization/bundles/en-US.json:2026]: Please add comments for all messages, these comments will help the agency understand the context of the text.
Thank you so much!

File: libs/sdk-ui/src/locales.ts [hunk 2]
```diff
@@ -16,4 +16,45 @@ export const messages: Record<string, MessageDescriptor> = defineMessages({
     t: { id: "visualization.numericValues.t" },
     p: { id: "visualization.numericValues.p" },
     e: { id: "visualization.numericValues.e" },
+    learnMore: { id: "configurationPanel.drillConfig.learn.more.link" },
```
Comment (khanhhuynh-gooddata) [inline libs/sdk-ui/src/locales.ts:19]: Could you add drillConfig prefix for this message?

File: libs/sdk-ui/src/base/localization/bundles/en-US.json (hunk 1 above)
Comment (phong-nguyen-duy) [reply libs/sdk-ui/src/base/localization/bundles/en-US.json]: done

File: libs/sdk-ui/src/locales.ts (hunk 2 above)
Comment (phong-nguyen-duy) [reply libs/sdk-ui/src/locales.ts]: done

File: libs/sdk-ui/src/base/localization/bundles/en-US.json (hunk 1 above)
Comment (khanhhuynh-gooddata) [reply libs/sdk-ui/src/base/localization/bundles/en-US.json]: There are some message missing comment: 
- Line: 2134 -> 2152
- Line: 2181

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/16768/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/18207/ : SUCCESS in 12m 01s

File: libs/sdk-ui/src/base/localization/bundles/en-US.json (hunk 1 above)
Comment (phong-nguyen-duy) [reply libs/sdk-ui/src/base/localization/bundles/en-US.json]: I think these messages are quite straightforward and comments are unnecessary

**Review by khanhhuynh-gooddata — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR4324

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/3445/ : SUCCESS in 9m 40s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/18208/ : SUCCESS in 12m 04s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/1066/ : SUCCESS in 10m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/4610/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/1325/ : FAILURE in 1m 13s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/1001/ : SUCCESS in 11m 09s

File: libs/sdk-ui/src/base/localization/bundles/en-US.json [hunk 3]
```diff
@@ -2015,5 +2015,190 @@
...
+        "comment": "Text for button add attribute bellow the current item",
+        "limit": 0
+    },
+    "attributeHierarchyEditor.howToWork.link": {
+        "value": "How to work with hierarchies",
+        "comment": "Link to document describe how attribute hierarchy works",
+        "limit": 0
+    },
+    "attributeHierarchyEditor.attribute.dropdown.title": {
+        "value": "Choose attribute ...",
```
Comment (xMort) [inline libs/sdk-ui/src/base/localization/bundles/en-US.json:2135]: There is no space between three dots in English.
Also, we use "…" single character symbol and comment should contain mention of this, for example `Use '…' as one character, not 3 dots ('...').` - see other such occurrences in SDK localization bundle.

File: libs/sdk-ui/src/base/localization/bundles/en-US.json [hunk 4]
```diff
@@ -2015,5 +2015,190 @@
...
+        "comment": "",
+        "limit": 0
+    },
+    "attributeHierarchyEditor.attribute.dropdown.search": {
+        "value": "Search ...",
```
Comment (xMort) [inline libs/sdk-ui/src/base/localization/bundles/en-US.json:2140]: There is no space between three dots in English.
Also, we use "…" single character symbol and comment should contain mention of this, for example `Use '…' as one character, not 3 dots ('...').` - see other such occurrences in SDK localization bundle.

File: libs/sdk-ui/src/base/localization/bundles/en-US.json [hunk 5]
```diff
@@ -2015,5 +2015,190 @@
...
+        "limit": 0
+    },
+    "attributeHierarchyList.tooltip.generic.date.info": {
+        "value": "This hierarchy is generated automatically and applies to any date dimension.",
+        "comment": "Text show in date hierarchy",
+        "limit": 0
+    },
+    "attributeHierarchyList.tooltip.hierarchy.levels": {
+        "value": "Hierarchy levels",
+        "comment": "",
```
Comment (xMort) [inline libs/sdk-ui/src/base/localization/bundles/en-US.json:2181]: Missing comment. Please check the other occurrences in the files as well.

**Pull request merged**